core/css.js

```diff
@@ -78,478 +78,273 @@ const inject = function(hasCss, pathToMedia) {
  */
 const CONTENT = [
```

Comment:
I understand why clang-format did this, but ooph...possibly we want to avoid it here?

Reply:
Yikes. I'm reverting that commit and I'll see if there's a better way to do this.
I'm thinking either to disable clang for these lines in some way or with a template string.